doc/upgrading.md

```diff
@@ -21,6 +22,40 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 * [Version 25](#version-25)
 * [Version 20](#version-20)
 
+## Version 34.0
+
+
+#### [RMB-885](https://issues.folio.org/browse/RMB-885)
+
+The following PostgresClient methods have been removed:
+
+```java
+
+<T> void streamGet(String table, T entity, String fieldName,
+   CQLWrapper filter, boolean returnIdField, String distinctOn,
+   Handler<T> streamHandler, Handler<AsyncResult<Void>> replyHandler)
+
+<T> void get(String table, Class<T> clazz, String fieldName, String where,
+   boolean returnCount, boolean returnIdField, boolean setId,
+   Handler<AsyncResult<Results<T>>> replyHandler)
+
+<T> void get(String table, Class<T> clazz, String fieldName, String where,
+   boolean returnCount, boolean returnIdField, boolean setId, List<FacetField> facets,
+   Handler<AsyncResult<Results<T>>> replyHandler)
+
+<T> void get(String table, Class<T> clazz, String fieldName, String where,
+   boolean returnCount, boolean returnIdField, boolean setId, List<FacetField> facets, String distinctOn,
+   Handler<AsyncResult<Results<T>>> replyHandler)
+
+<T> void get(String table, Class<T> clazz, String[] fields, String filter,
+   boolean returnCount, boolean setId,
+   Handler<AsyncResult<Results<T>>> replyHandler)
+
+<T> void get(String table, Class<T> clazz, String filter,
+   boolean returnCount, boolean setId,
+   Handler<AsyncResult<Results<T>>> replyHandler)
+```
```

Comment:
```suggestion
The `PostgresClient.streamGet` method without `PostgresClientStreamResult` parameter has been removed because it doesn't return totalCount and consumes too much memory. Use one of the other `PostgresClient.streamGet` methods with `PostgresClientStreamResult`.

`PostgresClient.get` methods with `String where` or `String filter` parameter have been removed to reduce SQL injection issues. Use `PostgresClient.get` methods with `CQLWrapper` or `Criterion` parameter instead.
```

Mentioning reasons and what to use instead.

The build will fail with a compile error showing where a removed method is called, no need to give the full details here.